libtsuba/include/tsuba/tsuba.h

```diff
@@ -67,16 +67,15 @@ KATANA_EXPORT katana::Result<void> Close(RDGHandle handle);
 /// \param name is storage location prefix that will be used to store the RDG
 KATANA_EXPORT katana::Result<void> Create(const std::string& name);
 
-/// RegisterIfAbsent registers a previously created RDG and attaches it to the
-/// namespace; infer the version by examining files in name. If the RDG is
-/// already registered, this operation is a noop.
-///
-/// \param name is storage location prefix that the RDG is stored in
-KATANA_EXPORT katana::Result<void> RegisterIfAbsent(const std::string& name);
-
-/// Forget an RDG, detaching it from the namespace
-/// \param name is storage location prefix that the RDG is stored in
-KATANA_EXPORT katana::Result<void> Forget(const std::string& name);
+struct KATANA_EXPORT RDGView {
+  uint64_t view_version{0};
+  std::string view_type;
+  std::string view_args;
+  std::string view_path;
+  uint64_t num_partitions{0};
+  uint32_t policy_id{0};
+  bool transpose{false};
+};
```

Comment:
Bye `RegisterIfAbsent`, I will not miss you.

Reply:
This looks like an important data structure.  Do we get doxygen comments for it?

Reply:
I will DEFINITELY not miss it either :)
I will add some comments.